tests/suite/dune

```diff
@@ -1,6 +1,6 @@
 (library
   (name test_suite)
-  (libraries str unix)
+  (libraries str unix fileutils)
```

Comment:
?

Reply:
`Sys.command("cp source dest")` is not portable on system like windows I think (?)
Fileutils should implement system commands like cp/rm etc... that are POSIX compatible